quotes-app-rs/.flox/env/manifest.toml

```
@@ -33,13 +33,35 @@ description = "A simple quotes server"
 version = "0.1.0"
 runtime-packages = ["libiconv", "gcc"]
 
+# This build vendors dependencies in a build with the sandbox off
+# so that we can use the network to fetch dependencies.
+[build.deps]
```

I've used prefixed names for the Go and Ruby projects that the intermediate packages and store paths have more recognisable names when switching between projects in this repo.
```suggestion
[build.quotes-app-rs-deps]
```